setup.py

```diff
@@ -7,6 +7,7 @@
     "description": "exports hardware related metrics",
     "use_scm_version": True,
     "setup_requires": ["setuptools_scm", "pyyaml"],
+    "install_requires": ["pyyaml", "pydantic", "prometheus_client"],
```

Comment:
we should refer to `requirements.txt` to avoid to have two place with same requirements.

Reply:
I am not quite sure about this either. I thought the `install_requires` and `requirement.txt` are for different purposes. https://stackoverflow.com/questions/14399534/reference-requirements-txt-for-the-install-requires-kwarg-in-setuptools-setup-py

Reply:
I think that in our case we do not take advantage of that, so I do not see point to have same requirement in two places and if you want to change one you will need to change it in two places. For us requirements in requirements.txt are minimal requirements needed for these to work.